__tests__/run.test.ts

```diff
@@ -0,0 +1,7 @@
+import {run} from '../src/login'
+
+describe('This is a placeholder for intial test cases, to be removed', () => {
+    test('Dummy test case', async () => {
+        await expect(run()).rejects.toThrow();
```

Comment:
Is this because you didn't give any inputs?

Reply:
yep